src/js/helpers/elementCreatorHelper.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * A series of functions to assist in creating DOM elements
+ *
+ * Add more as needed & make sure to list them here :
+ *
+ * - createTitle
+ * - createContainer
+ */
```

Comment:
Add the createButton function to the glossary
```suggestion
- createButton
 */
```